standup_notes/resources/standup-notes.bash

```diff
@@ -2,36 +2,32 @@
 
 _standup_completions()
 {
-    local cur prev opts
+    local cur prev opts prev prev2
```

Comment:
do you mean to have "prev" twice?

Reply:
Yes I do, prev is used for the previous command while prev2 is used to check to commands back so if the user wants to copy previous days notes,they are not prompted with extra options

Reply:
Looks like you define "prev" twice though, and "prev2" once.

Reply:
ah, removed extra prev